src/services/status.service.js

```diff
@@ -1,30 +1,145 @@
+/**
+ * Functions for getting and setting status values
+ * @module services/status
+ */
+
 const {
   getStoredStatus,
-  updateStoredStatus
+  updateStoredStatus,
+  getEmailDistribution
 } = require("../connectors/status/status-db.connector");
 
+const { logEmitter } = require("./logging.service");
+const {
+  NOTIFY_STATUS_TEMPLATE,
+  ENVIRONMENT_DESCRIPTION
+} = require("../config");
+const { sendSingleEmail } = require("../connectors/notify/notify.connector");
+
+/**
+ * Function that returns the status
+ *
+ * @param {string} statusName The name of status field to return
+ *
+ * @returns {object} All status values for the specified field name
+ */
 const getStatus = async statusName => {
+  logEmitter.emit("functionCall", "status.service", "getStatus");
+
   const status = await getStoredStatus();
+
+  logEmitter.emit("functionSuccess", "status.service", "getStatus");
+
   return statusName ? status[statusName] : status;
 };
 
+/**
+ * Updates a specified status value
+ *
+ * @param {string} statusName The status field name
+ * @param {any} newStatus The new status value
+ *
+ * @returns {any} The new status value
```

Comment:
this too

Reply:
This could also potentially be any type (e.g. Int, Bool, Double)